shapecorners.cpp

```diff
@@ -71,15 +71,20 @@ ShapeCornersEffect::ShapeCornersEffect() : KWin::Effect(), m_shader(nullptr)
     if (file.open(QFile::ReadOnly))
     {
         QByteArray frag = file.readAll();
-        m_shader = KWin::ShaderManager::instance()->generateCustomShader(KWin::ShaderTrait::MapTexture, QByteArray(), frag);
+        auto shader = KWin::ShaderManager::instance()->generateCustomShader(KWin::ShaderTrait::MapTexture, QByteArray(), frag);
+#if KWIN_EFFECT_API_VERSION >= 235
```

Comment:
You could better use QT_VERSION_CHECK, that does the bitshifting for you

Reply:
Can you elaborate?

Reply:
Oh nevermind, KWin does the versioning really weird :/